crates/move-model/src/model.rs

```diff
@@ -1996,6 +2007,7 @@ impl GlobalEnv {
             self.object_table_is_empty_qid(),
             self.object_table_destroy_empty_qid(),
             self.object_table_value_id_qid(),
+            self.object_borrow_uid_qid(),
```

Comment:
is `object::borrow_uid` native? If yes, then it's not necessary to add it here. This is a list of "intrinsic" functions, basically functions that aren't native but we want the prover to treat them as native.